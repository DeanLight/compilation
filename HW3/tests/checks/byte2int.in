//byte to int

void main()
{
	int ii;
	byte bb;
	{
		int z;
	}
	ii = bb;
	bb = ii;
}