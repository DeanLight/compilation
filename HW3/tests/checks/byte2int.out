---end scope---
z INT 2
line 11: type mismatch
